learning-another-language/our-path.md

```diff
@@ -0,0 +1,112 @@
+# Our Path
+
+## Learning Out of Necessity
+
+Jing has been working on a Python project with their team for the last few months. They've had ups and downs, but the project is getting deployed. They feel very proud of their work.
+
+However, Jing's manager just called them into a meeting. There's a new opportunity and the manager is recommending Jing to join this new team! It's a great opportunity for them. However, there's one huge concern: Jing's new team will build software in JavaScript.
+
+Jing has never seen JavaScript code before, much less written it!
+
+How can Jing effectively learn another programming language?
+
+How can _we_ most effectively learn another programming language?
+
+## Our Approach
+
+This curriculum will take the following approach to learning JavaScript:
+
+1. Write Hello, World!
+1. Write FizzBuzz
+1. Summarize what we've learned so far
+1. Establish resources
+
+From there, we will make a list of a few major concepts that are unique to JavaScript.
+
+At the end of these steps, we should have enough knowledge of JavaScript that lets us feel comfortable playing around with the language.
+
+### Write Hello, World!
+
+Hello, World! is a great exercise to start with because it forces us to learn how to:
+
+- run Javascript code
+- write a print statement
+- read the output
+- debug errors
+
+### Write FizzBuzz
+
+FizzBuzz will force us how to:
+
+- use variables
+- use conditionals
+- use equality-checking and comparison
+- define functions
+- use parameters
+- use `return`
+- call functions
+
+### Take Notes and Summarize
+
+After jumping into the code and picking up a lot of syntax, we'll need to pause, reflect, and summarize our learning so far.
+
+### Establish Resources
+
+The next step is deciding which resources are on our side. What does a "good," trustworthy resource look like?
+
+### Collect More Topics to Learn
+
+For our JavaScript curriculum, we'll dive into these topics:
+
+- What's up with semicolons?
+- Why do variables look like that?
+- What is an object? Where are Python dictionaries?
+- Why are there so many different syntaxes to write functions? And how are they different?
+
+### Decide When to Start Solving Problems
+
+One final, but extremely important step to take is deciding when to move past syntax learning.
+
+At some point, it will be more effective to learn JavaScript by encountering problems.
+
+This moment comes before we feel ready. 😅🙂
+
+## Change in Study Style
+
+When learning another programming language, we should allow ourselves to be more exploratory.
+
+At many times, we will _choose_ to:
+
+- Learn a bit about something, and then move on
+- Focus on syntax and muscle memory
+- Take notes for things to revisit and learn more about
+
+This requires trust in ourselves!
+
+In the moment of coding, there may be something we get stuck on, and we need to trust that if we keep pushing, it will come together.
```

Comment:
the full stop hear sounds a but more resolute to me
```suggestion
In the moment of coding, there may be something we get stuck on. We need to trust that if we keep pushing, it will come together.
```